docs/meshcloud.index.md

```diff
@@ -2,56 +2,68 @@
-| [meshPlatform](meshcloud.platforms.md) | An individual cloud platform connected to meshcloud.                                      |
-| [meshLocation](meshcloud.platforms.md) | A grouping of meshPlatforms, e.g. by geographic region.                                   |
-
+Welcome to the meshStack documentation! This part of the documentation is intended for end users
+of the meshStack platform. Be aware that meshStack might be rebranded in your organization and has a different name,
+but everything you find in the documentation will apply to the meshStack you are using as well.
+
+meshStack is useful for anyone that wants to use cloud services in their organization and manage their costs.
+Maybe you are a DevOps team that wants to create a few cloud projects in Microsoft Azure their projects across
+one or more cloud providers, or maybe you are a Project Manager that wants to manage the budget of a cloud project.
+
+The actual cloud platforms and services that are available to you depend on the setup of the meshStack and your organization.
+One organization might only offer Microsoft Azure, but another might offer all major cloud providers. Your meshStack will automatically
+reflect whatever is available in your case.
+
+## Concepts
+
+To work with meshStack, the following concepts are of importance:
+
+- [Workspace](./meshcloud.workspace.md): The workspace is the highest level that you will work with. In here, you will invite
+  your entire team of managers, developers and more. In the workspace you will create one or more projects.
+- [Project](./meshcloud.project.md): The project is a cloud-agnostic concept which usually maps to an environment of an application
+  that you and your team are developing. The project will have a payment method assigned that acts as the budget. On the project
+  you invite team members that are needed for that environment, e.g. for a production project you will invite your SRES. Within
+  the project, you will have one or more cloud tenants.
+- [Tenant](./meshcloud.tenant.md): The tenant is an isolated and tangible place in an actual cloud provider. It depends on what cloud
+  you use what this maps to. For example for AWS this maps to an `Account`, for Azure this is a `Subscription` and for Google Cloud
+  it is a `Project`. meshStack can easily let you log in and redirect you to the tenant in the respective cloud provider.
+- [Landing Zone](./meshcloud.landing-zones.md): The landing zone is a template created by your organization that you pick
+  when you create a new tenant. It cannot be changed afterward. The idea of the landing zone is that you already get a set
+  of policies and perhaps resources to start using the cloud right away according to the compliance and best practices
+  of your organization.
+- [Building Blocks](./administration.building-blocks.md): A building block is a piece of infrastructure that can be booked
```

Comment:
I wonder if this is always infrastructure

Reply:
It is based on Terraform and Terraform is an infrastructure-as-code tool so I think it's the best explanation we can have,